### PR TITLE
Added satellite-maintain repo to capsule

### PR DESCRIPTION
**Pull request by san7ket**

### CODE REVIEW

File: upgrade/capsule.py [hunk 1]
```diff
@@ -136,6 +137,9 @@ def satellite6_capsule_upgrade(cap_host, sat_host):
         major_ver, from_version))
     if from_version == '6.1' and major_ver == '6':
         enable_repos('rhel-server-rhscl-{0}-rpms'.format(major_ver))
+    if to_version in ['6.4']:
+        # setup foreman-maintain
+        setup_foreman_maintain()
```
Comment (jyejare) [inline upgrade/capsule.py]: s/current/```setup_foreman_maintain() if to_version in ['6.4']```/

Comment (jyejare) [reply upgrade/capsule.py]: @san7ket ,
Do thisL
```
hi() if to_ver in ['6.4'] else None
hi sanket
```

**Comment by ntkathole** [comment]
@san7ket we need same for satellite as well while using traditional path.

File: upgrade/capsule.py (hunk 1 above)
Comment (ntkathole) [reply upgrade/capsule.py]: Will suggest to go for to_ver not in ['6.1', '6.2', '6.3']

**Comment by san7ket** [comment]
@ntkathole addressed

**Pull request merged**